src/manual/peer2peer/extension/background.js

```diff
@@ -44,3 +48,8 @@ function cancelScreenSharing() {
     chrome.desktopCapture.cancelChooseDesktopMedia(desktopMediaRequestId);
   }
 }
+
+function getChromeVersion() {
+  var raw = navigator.userAgent.match(/Chrome\/([0-9]+)\./);
```

Comment:
It's an NPM dependency, I added a link in the comment, you can just copy the code in here. Link again: https://github.com/webrtc/adapter/blob/master/src/js/utils.js#L75